FirebaseUI.xcodeproj/project.pbxproj

```diff
@@ -1035,163 +1132,194 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
-		072ABD579ED6CE7C8FC98658 /* 📦 Check Pods Manifest.lock */ = {
+		072ABD579ED6CE7C8FC98658 /* [CP] Check Pods Manifest.lock */ = {
```

Comment:
pod-deintegrate doesn't delete those. You can simply delete those 3 build phases.